### PR TITLE
fix: examples added, authors section modified

### PR DESCRIPTION
**Pull request by fess-v**

### CODE REVIEW

File: sips/sip-02x/sip-02x-non-sequential-multisig-transactions.md [hunk 1]
```diff
@@ -55,6 +55,22 @@ signature_1(tx), signature_2(tx), ..., signature_n(tx)
...
+- Once the 5th member has signed a transaction, further signatures are restricted.
+
+This leads to a problem where members positioned earlier in the sequence possess more flexibility compared to those later in the order, for instance:
+
+- Initiating a transaction by the 4th or 5th member is impractical, as there are insufficient subsequent members to complete the signing process.
+- In a scenario where the 1st member initiates a transaction and the 4th signs it, it restricts the 2nd and 3rd members from signing. The responsibility then falls solely on the 5th member to finalize the transaction. This becomes cumbersome if, for instance, the 5th member is unavailable, forcing others to await their signature.
+- Similarly, if the 3rd member both initiates and signs a transaction, it requires signatures from both the 4th and 5th members. The 1st and 2nd members are sidelined unless a new transaction sequence is initiated.
+
+To summarize, while such a multisig setup might suffice for smaller teams, its effectiveness dwindles as the number of signatories increases, especially with four or more participants. It complicates transaction initiation and obtaining the needed signatures.
+
```
Comment (jbencin) [inline sips/sip-02x/sip-02x-non-sequential-multisig-transactions.md]: A couple points:
 - This is a bit repetitive
 - It could be ordered better
 - When writing Markdown use one sentence per line, it makes for better diffs (I sometimes forget also)

Here's my suggestion. I re-ordered, removed some stuff, and re-worded a few things:

```suggestion
## Examples

Imagine a DAO that has a management team comprised of five members.
They create a 3 out of 5 multisig account on Stacks.
The existing multisig standard mandates that all transactions from this account be signed in an order which is established upon account creation.
The ordering requirement creates a hierarchy where keys near the start of the sequence have more flexibility than those near the end.
To illustrate some of the limitations this creates:

- In a scenario where the 1st member initiates a transaction and the 4th signs it, it prohibits the 2nd and 3rd members from signing. The responsibility then falls solely on the 5th member to finalize the transaction.
- Once the 5th member has signed a transaction, no further signatures are possible.
- If the 3rd member initiates a transaction, only the 4th and 5th members are eligible to provide subsequent signatures.
- Initiating a transaction by the 4th or 5th member is impossible, as there are insufficient subsequent members to complete the signing process.

To summarize, while such a multisig setup might suffice for smaller teams, as the number of required signers increases, it becomes increasingly difficult to create a transaction. This SIP aims to remove these limitations.
```
